### ISSUE TITLE
Transfer Learning

### ISSUE THREAD

**Issue by imnimn** [seongmin-kye/meta-SR#3]
Want to thank you very much for sharing your code, my first tests showed good results 
I want to ask if you could give a hint in transfer learning the shared pre-trained models to achieve similar results in other languages?

Best wishes,

**Comment by seongmin-kye** [comment]
Hi. Thank you for your attention!
What does language exactly mean? Is that program language or literal language such as Korean, English?
If it is the latter, it is important to match the feature extract setting (40-mel filterbank + mean normalization).
Secondly, input acoustic feature must be matched with the paper (crop size). Lastly, it would be nice to experiment with various adjustments of the initial learning rate.

If it helped, it would be good to give a star.  :)
Thanks.

**Comment by imnimn** [comment]
Thank you for the helpful quick reply
It is for literal language, for English
I will try that and inform you about the results
